introduction.md

```diff
@@ -1 +1,3 @@
-# Hi, nice to meet you, I'm Jordan! I enjoyed the Electives path the most.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?